### PR TITLE
✨ Source Appsflyer: Add JWT Support to appsflyer source

### PR DESCRIPTION
**Pull request by anjshrg**
## What
*Describe what the change is solving*
Appsflyer has deprecated v1 API token since 31st Aug and introduced V2 which is a JWT based token authentication. This change modifies existing streams to support the same.
*It helps to add screenshots if it affects the frontend.*

## How
*Describe the solution*
It uses JWT based bearer authentication.

## 🚨 User Impact 🚨
*Are there any breaking changes? What is the end result perceived by the user?*
No, Nothing will break for end user. As auth method has been upgraded at connector level.

## Pre-merge Actions
*Expand the relevant checklist and delete the others.*

<details><summary><strong>Updating a connector</strong></summary>

### Community member or Airbyter

- Grant edit access to maintainers ([instructions](https://docs.github.com/en/github/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork#enabling-repository-maintainer-permissions-on-existing-pull-requests))
- Unit & integration tests added

### Airbyter

If this is a community PR, the Airbyte engineer reviewing this PR is responsible for the below items.

- Create a non-forked branch based on this PR and test the below items on it
- Build is successful
- If new credentials are required for use in CI, add them to GSM. [Instructions](https://docs.airbyte.io/connector-development#using-credentials-in-ci).

</details>

</details>

### CODE REVIEW

**Comment by anjshrg** [comment]
Here is the article regarding deprecating v1 token:
https://support.appsflyer.com/hc/en-us/articles/12399683708305-Bulletin-API-token-changes

File: airbyte-integrations/connectors/source-appsflyer/metadata.yaml [hunk 1]
```diff
@@ -2,7 +2,7 @@ data:
   connectorSubtype: api
   connectorType: source
   definitionId: 16447954-e6a8-4593-b140-43dea13bc457
-  dockerImageTag: 0.1.0
+  dockerImageTag: 0.1.1
```
[the remark this answers is not in the document]
Comment (marcosmarxm) [reply airbyte-integrations/connectors/source-appsflyer/metadata.yaml:5]: Also update the Dockerfile version.

**Review by marcosmarxm — CHANGES_REQUESTED** [review]

File: airbyte-integrations/connectors/source-appsflyer/metadata.yaml (hunk 1 above)
[the remark this answers is not in the document]
Comment (anjshrg) [reply airbyte-integrations/connectors/source-appsflyer/metadata.yaml]: Addressed

**Comment by anjshrg** [comment]
@marcosmarxm have made necessary changes and tested locally. Kindly take a look

**Comment by sh4sh** [comment]
Bot was auto-formatting files unrelated to this PR so I hard reset. Sorry for the spam yall.

**Review by marcosmarxm — COMMENTED** [review]
Thanks for the contribution @anjshrg I'll review during this week or next one. Please let me know if you have any further questions about the contribution

**Comment by anjshrg** [comment]
@marcosmarxm please let me know if anything else is required to be completed at my end.

**Comment by marcosmarxm** [comment]
The latest version of the connector now supports API v2. Please check it @anjshrg sorry for the long delay without any updates here but I decided to close the contribution. Please check latest version of the connector and if the issue is still there let me know to work together and reopen this.

**Pull request closed without merging**